interfaces/INameWrapper.sol

```diff
@@ -13,7 +13,7 @@ uint96 constant CANNOT_REPLACE_SUBDOMAIN = 64;
 uint96 constant CAN_DO_EVERYTHING = 0;
 
 interface INameWrapper is IERC1155 {
-    enum ParentVulnerability {Safe, Registrant, Controller, Fuses, Expired}
+    enum NameSafety {Safe, Registrant, Controller, Fuses, Expired}
```

Comment:
Given the new name, some of the fields might need to be more specific - maybe `RegistrantNotWrapped`, `ControllerNotWrapped`, and `SubdomainReplacementAllowed`?